.github/workflows/push-pr-ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI on push or pull request for python 3.8
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.8"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest
```

Comment:
pytest is part of the requirements so we don't need to install it separately

Reply:
Fixed the comment as part of [commit](https://github.com/pylabel-project/pylabel/pull/8/commits/5db3cd09bda02686306e2dcccbd6e00f9d75892b)